Client/Coordinators/Router/Route.swift

```diff
@@ -0,0 +1,90 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+/**
+ An enumeration representing different navigational routes in an application.
+ */
```

Comment:
We try to use the `///` format for comments instead of `/** */`, would you mind using those instead? 🙏 I normally use the keyboard shortcut `option + cmd + /`

Reply:
No problem, I can revert the change. I actually used a tool myself to generate this format so I don't mind changing it.